src/app_model/backends/qt/_qaction.py

```diff
@@ -54,6 +56,11 @@ def _on_triggered(self, checked: bool) -> None:
         # to raise any exceptions.
         self._app.commands.execute_command(self._command_id).result()
 
+    def setToolTip(self, tooltip: str) -> None:
```

Comment:
with this pattern, is there any way for someone to opt *out* of this behavior?  For example, if they wish to set their own tooltip (sans suffix).  What do you think about simply calling `self.setToolTip` with the text and the shortcut suffix in `__init__` (and then later calls to `setToolTip()` would override it).  ... or some other method that makes it opt-out-able

Reply:
Oh I see, maybe I overdid a little bit here trying to implement a way to always show the keybinding 😅 Will simplify things then :+1:

Also, regardless of the simplification. could it make sense to add over the `Action` definition a field like the one for `CommandRule` definition for the icon visibility?

https://github.com/pyapp-kit/app-model/blob/4afa6f6fcf046bb7e48aa91f88f21cefc5b2352b/src/app_model/types/_command_rule.py#L61-L65

So something like `keybinding_visible_in_tooltip` and make its default value `False`?

Reply:
i think i'd prefer not to have that sort of "view-specific" setting in the model itself.